packages/serverless-logic-web-tools/src/homepage/recentModels/WorkspacesTableRow.tsx

```diff
@@ -56,12 +56,14 @@ export function WorkspacesTableRow(props: WorkspacesTableRowProps) {
 
   const linkTo = useMemo(
     () =>
-      routes.workspaceWithFilePath.path({
-        workspaceId: editableFiles[0].workspaceId,
-        fileRelativePath: editableFiles[0].relativePathWithoutExtension,
-        extension: editableFiles[0].extension,
-      }),
-    [editableFiles]
+      !isWsFolder
+        ? routes.workspaceWithFilePath.path({
+            workspaceId: editableFiles[0].workspaceId,
+            fileRelativePath: editableFiles[0].relativePathWithoutExtension,
+            extension: editableFiles[0].extension,
+          })
+        : routes.workspaceWithFiles.path({ workspaceId: editableFiles[0].workspaceId }),
```

Comment:
```suggestion
        : routes.workspaceWithFiles.path({ workspaceId }),
```
Can you reuse `workspaceId` from line 53?